exotica_python/src/pyexotica.cpp

```diff
@@ -1097,6 +1107,10 @@ PYBIND11_MODULE(_pyexotica, module)
         return vec;
     });
 
+    // KinematicElement
+    // TODO: bindings for methods
+    py::class_<KinematicElement, std::shared_ptr<KinematicElement>>(kin, "KinematicElement");
```

Comment:
I prefer to keep this class out of the Python API. It opens up the code to a lot of low level trouble.

Reply:
... and it may change when we turn `KinematicTree` into a plug-in to open up for other kinematics solvers

Reply:
I exposed it mainly for the `get_tree_map`. If `get_tree_map` changes it's function to just provide the names as get_tree_names, exposing the `KinematicElement` is not required anymore and will be removed.